format/private/format.sh

```diff
@@ -122,7 +119,22 @@ function ls-files {
           fi
           find_args+=("-name" "${patterns[$i]}")
         done
-        find "$@" "${find_args[@]}"
+        files=$(find "$@" "${find_args[@]}")
+    fi
+
+    if [[ $disable_git_attribute_checks == true ]]; then
+      echo $files
```

Comment:
This doesn't seem to be right because the call site expects line-separated list.

Reply:
I think it is due to `echo "$files" | tr \\n \\0`.